### DISCUSSION TITLE
request for enhancement: cloned project shall get user rights of parent

### DISCUSSION THREAD

**Discussion by JoergBruenner** [DependencyTrack/dependency-track#2224] [Ideas]
Hi,

automating import of new BoM versions lead to a lot scripting effort: You have to collect user and team rights
of the parent to bring these to the new cloned project. How about introducing a new option beside
- Include tags
- Include properties
- Include components
- Include services
- Include audit history?
Lets call it "Include user rights". If enabled, the cloned project gets the complete set of rights of its parent.

What do you think?

Jörg

**Comment by stevespringett** [comment]
This is coming in v4.7. See #1534

**Comment by JoergBruenner** [comment]
Great!Von meinem/meiner Galaxy gesendet
-------- Ursprüngliche Nachricht --------Von: Steve Springett ***@***.***> Datum: 02.12.22  17:36  (GMT+01:00) An: DependencyTrack/dependency-track ***@***.***> Cc: JoergBruenner ***@***.***>, Author ***@***.***> Betreff: Re: [DependencyTrack/dependency-track] request for enhancement:
  cloned project shall get user rights of parent (Discussion #2224) 
This is coming in v4.7. See #1534
